src/openwebui/tools/run_code.py

```diff
@@ -269,6 +269,11 @@ def _log(filename: str, log_line: str):
                         done=True,
                         description=f"[DEBUG MODE] status={status}; output={output}; valves=[{valves}]; debug={per_file_logs}",
                     )
+
+            await emitter.message(
+                f"\n<details>\n<summary>Code Execution</summary>\nI executed the following {language} code:\n```{language}\n{code}\n```\n```Output\n{output}\n```\n</details>\n"
+            )
```

Comment:
I think this message should only be emitted if the code execution was successful. As currently written, the message is emitted in all cases, including the error cases. Please either move this to the success-case branch (line 254), or leave this here but check `if status == "OK"`.

Reply:
Okay, I have completed the revision, but it seems that it would look better to return this content at any time, or should we use a separate foldable message block for errors?

Reply:
The error is already displayed in the "status" part of the chat message, not sure there's any value in adding it as a real part of the message.